src/measure_theory/group/prod.lean

```diff
@@ -40,33 +40,42 @@ variables (G : Type*) [measurable_space G]
 variables [group G] [has_measurable_mul₂ G]
 variables (μ ν : measure G) [sigma_finite ν] [sigma_finite μ] {s : set G}
 
-/-- The map `(x, y) ↦ (x, xy)` as a `measurable_equiv`. This is a shear mapping. -/
-@[to_additive "The map `(x, y) ↦ (x, x + y)` as a `measurable_equiv`.
-This is a shear mapping."]
+/-- The map `(x, y) ↦ (x, xy)` as a `measurable_equiv`. -/
+@[to_additive "The map `(x, y) ↦ (x, x + y)` as a `measurable_equiv`."]
 protected def measurable_equiv.shear_mul_right [has_measurable_inv G] : G × G ≃ᵐ G × G :=
 { measurable_to_fun  := measurable_fst.prod_mk measurable_mul,
   measurable_inv_fun := measurable_fst.prod_mk $ measurable_fst.inv.mul measurable_snd,
   .. equiv.prod_shear (equiv.refl _) equiv.mul_left }
 
+/-- The map `(x, y) ↦ (x, yx)` as a `measurable_equiv` with as inverse `(x, y) ↦ (x, y / x)` -/
+@[to_additive
+  "The map `(x, y) ↦ (x, y + x)` as a `measurable_equiv` with as inverse `(x, y) ↦ (x, y - x)`."]
+protected def measurable_equiv.shear_mul_left [has_measurable_inv G] : G × G ≃ᵐ G × G :=
```

Comment:
I'm not a fan of this name, I would expect this to mean the same as above but on the left component instead. `antishear` is probably very bad too, but maybe something along these lines could work.

Reply:
I agree that this name is not very good, I'm happy to modify it if you have better suggestions. I personally don't like `antishear`.

Reply:
`shear_div_right`?

Reply:
@sgouezel That name makes more sense if I invert the equivalence. I think I'll do that.